service/src/main/java/bio/terra/profile/app/configuration/WebConfig.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.profile.app.configuration;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.resource.TransformedResource;
+
+@Component
+public class WebConfig implements WebMvcConfigurer {
```

Comment:
This class intercepts swagger-ui web calls, and makes replacements to support B2C OAuth. (Borrowed approach from TDR.)